packages/opentelemetry-node/lib/instrumentations.js

```diff
@@ -180,7 +206,39 @@ function getInstrumentations(opts = {}) {
     /** @type {Array<Instrumentation>} */
     const instrumentations = [];
 
+    const getInstrName = (name) => `@opentelemetry/instrumentation-${name}`;
+    const enabledFromEnv = getEnvVar('OTEL_NODE_ENABLED_INSTRUMENTATIONS');
+    const disabledFromEnv = getEnvVar('OTEL_NODE_DISABLED_INSTRUMENTATIONS');
+    const enabledList = parseStringList(enabledFromEnv).map(getInstrName);
+    const disabledList = parseStringList(disabledFromEnv).map(getInstrName);
+
+    for (const name of enabledList) {
+        if (!INSTRUMENTATIONS[name]) {
+            log.warn(
+                `Invalid instrumentation "${name}" specified in the environment variable OTEL_NODE_ENABLED_INSTRUMENTATIONS`
+            );
+        }
+    }
+    for (const name of disabledList) {
+        if (!INSTRUMENTATIONS[name]) {
+            log.warn(
+                `Invalid instrumentation "${name}" specified in the environment variable OTEL_NODE_DISABLED_INSTRUMENTATIONS`
+            );
+        }
+    }
+
     Object.keys(INSTRUMENTATIONS).forEach((name) => {
+        // 1st check what was configured via env
+        if (disabledList.length && disabledList.includes(name)) {
+            return;
+        }
+
+        if (enabledList.length && !enabledList.includes(name)) {
+            return;
+        }
+
+        // 2nd check the configuration passed
+        // XXX: discuss what takes precedence if `opts` argument id passed in
```

Comment:
note for reviewer: leave your thoughts here :)

Reply:
if I got it right form our last meeting I've done the changes 
- env vars take precedence over config
- if env var enables a instrumentation disabled by config it becomes enabled